src/modules/core/components/avatar-select-mobile/avatar-select-mobile.stories.ts

```diff
@@ -0,0 +1,16 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { AvatarSelectMobile } from './avatar-select-mobile';
+
+const meta: Meta<typeof AvatarSelectMobile> = {
```

Comment:
Maybe we should remove changing background color when user on mobile press this area, it creates flickering.

Reply:
done by adding: 

```scss
* {
  -webkit-tap-highlight-color: transparent;
  -moz-tap-highlight-color: transparent;
}
```

It also removes same blink for other touchable elements (e.g. buttons).